_topic_maps/_topic_map.yml

```
@@ -340,6 +340,20 @@ Topics:
     File: installing-ibm-power
   - Name: Restricted network IBM Power installation
     File: installing-restricted-networks-ibm-power
+- Name: Installing on IBM Power VS
+  Dir: installing_ibm_power
```

This puts the PowerVS docs in the same directory as Power (on-premise).  As the equivalent of cloud IPI compared to baremetal UPI, these are handled as different platforms and should be separated as such, e.g.:
```suggestion
  Dir: installing_ibm_powervs
```
and the new files in `installing/installing_ibm_power` moved to the new directory etc.